src/App.js

```diff
@@ -2,24 +2,29 @@ import React from 'react';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    time: new Date(),
+  }
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+  componentDidMount() {
+    setInterval(() => {
+      this.setState({
+        time: new Date(),
+      });
+    }, 1000);
+  }
 
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+  render() {
+    return (
+      <div className="App">
+        <div className="clock">
+          {this.state.time.toLocaleTimeString().split(':').join(' : ')}
+          <div className="strip" key={Math.random()} />
```

Comment:
```suggestion
          <div className="strip" />
```
- since this is not .map of an array of the elements, you don't need key there
- Math.random it's a bad choice for the key